src/Form/Helpers/FormService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Form\Helpers;
+
+use MyParcelNL\Pdk\Settings\View\CarrierSettingsView;
+use MyParcelNL\Pdk\Settings\View\CustomsSettingsView;
+use MyParcelNL\Pdk\Settings\View\DeliveryOptionsStringsSettingsView;
+use MyParcelNL\Pdk\Settings\View\GeneralSettingsView;
+use MyParcelNL\Pdk\Settings\View\LabelSettingsView;
+use MyParcelNL\Pdk\Settings\View\OrderSettingsView;
+
+class FormService
+{
+    /**
+     * @param  string     $view
+     * @param  null|array $data
+     *
+     * @return array
+     */
+    public function getForm(string $view, ?array $data): array
```

Comment:
Dit is niet goed, nu worden altijd nog steeds alle views gegenereerd (met foute data), en kun je er niet meerdere tegelijk ophalen. En die strings zijn magisch. Kun je hier geen betere structuur voor bedenken?